### PR TITLE
Add code of conduct and news

### LINKED ISSUE #188: Adding CODE_OF_CONDUCT.md
https://github.com/insightsengineering/rtables/blob/main/.github/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by Melkiades**
Fixes #188

### CODE REVIEW

**Comment by shajoezhu** [comment]
hi @Melkiades 

can you add some words on using staged dependencies in the contirbution.md file too. thanks!

https://github.com/insightsengineering/rtables/issues/803

**Comment by Melkiades** [comment]
> hi @Melkiades
> 
> can you add some words on using staged dependencies in the contirbution.md file too. thanks!
> 
> [insightsengineering/rtables#803](https://github.com/insightsengineering/rtables/issues/803)

I checked and it is identical to {tern}. It has also the staged.deps bit ;)

File: README.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# rlistings <a href='https://github.com/insightsengineering/rlistings'><img src="man/figures/logo.png" align="right" height="200" width="200"/></a>
+# rlistings
 
 <!-- start badges -->
+
 [![Check
-🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://insightsengineering.github.io/rlistings/main/unit-test-report/)
+🛠](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml/badge.svg)](https://github.com/insightsengineering/rlistings/actions/workflows/check.yaml)
```
Comment (Melkiades) [inline README.md:9]: I just rebuilt it. I guess it was wrongly changed here instead that the quarto Rmd. Should I put them back?

Comment (shajoezhu) [reply README.md:9]: this link was right, right? https://insightsengineering.github.io/rlistings/main/unit-test-report/

@pawelru @cicdguy

**Comment by shajoezhu** [comment]
> > hi @Melkiades
> > can you add some words on using staged dependencies in the contirbution.md file too. thanks!
> > [insightsengineering/rtables#803](https://github.com/insightsengineering/rtables/issues/803)
> 
> I checked and it is identical to {tern}. It has also the staged.deps bit ;)

you are right, somehow i remembered wrong. I will close the issue in rtables too. Thanks for checking!

**Comment by shajoezhu** [comment]
> > > hi @Melkiades
> > > can you add some words on using staged dependencies in the contirbution.md file too. thanks!
> > > [insightsengineering/rtables#803](https://github.com/insightsengineering/rtables/issues/803)
> > 
> > 
> > I checked and it is identical to {tern}. It has also the staged.deps bit ;)
> 
> you are right, somehow i remembered wrong. I will close the issue in rtables too. Thanks for checking!

hi @Melkiades , actually, now i remembered, we need to add a sentence for contributors, on how to use staged.dependencies, and stop the downstream checks. 

otherwise the installation is not smooth for getting the right branches. i think we will need to add this in all

formatters, rlistings, rtables and tern

File: README.md (hunk 1 above)
Comment (shajoezhu) [reply README.md:9]: Thanks @cicdguy , i am a bit inclined to keep the unittest-test-report @Melkiades

**Comment by Melkiades** [comment]
> > > > hi @Melkiades
> > > > can you add some words on using staged dependencies in the contirbution.md file too. thanks!
> > > > [insightsengineering/rtables#803](https://github.com/insightsengineering/rtables/issues/803)
> > > 
> > > 
> > > I checked and it is identical to {tern}. It has also the staged.deps bit ;)
> > 
> > 
> > you are right, somehow i remembered wrong. I will close the issue in rtables too. Thanks for checking!
> 
> hi @Melkiades , actually, now i remembered, we need to add a sentence for contributors, on how to use staged.dependencies, and stop the downstream checks.
> 
> otherwise the installation is not smooth for getting the right branches. i think we will need to add this in all
> 
> formatters, rlistings, rtables and tern

It is not super clear to me what you want to add; I would do it in a different PR once we have created the issues ^^

**Review by shajoezhu — APPROVED** [review]
lgtm! thanks @Melkiades lets merge this in, and get the staged.deps in a seperate issue

**Pull request merged**